Sources/CocoaLumberjack/DDLog.m

```diff
@@ -462,11 +462,13 @@ - (void)log:(BOOL)asynchronous
    function:(const char *)function
        line:(NSUInteger)line
         tag:(id)tag {
+    NSString* fileString = @(file);
+    NSAssert(fileString, @"Invalid file parameter");
```

Comment:
Added this assertion to catch both `file == nil` and `file` is invalid UTF8 string.

Reply:
Isn't `initWithMessage:` a more logical place for that assertion? Otherwise it's still possible to create an invalid `DDLogMessage` using its initializer directly.